README.md

```diff
@@ -33,6 +33,55 @@ Consigliamo l'utilizzo dei test per migliorare la stabilità del codice per quel
 Utilizzare questa sintassi `[ref #<numero_card> :] <titolo>`
 - Ogni commit di sviluppo dovrebbe contenere il riferimento per la card di sviluppo
 
+## Dotenv-Vault e Condivisione Envfiles
+
+Dotenv-vault è lo strumento che utilizziamo per la condivisione e il monitoraggio degli envfile.
+Gli envfile sono spesso necessari nella fase di start di un progetto e provvedono a fornire i valori per le variabili d'ambiente relative al progetto stesso. Tuttavia gli envfile contengono spesso dati sensibili ed è quindi necessario aggiungere un layer di sicurezza alla condivisone di tali informazioni.
+
+**Perchè l'abbiamo scelto?**
+
+- Ci aiuta a garantire gli standard di sicurezza ad un costo operativo minimo.
+- Facilita il lavoro di team, è difatti possibile gestire gli accessi ai singoli "spazi di lavoro", detti "Vault", tramite Dotenv-vault
+
+Nel suo ciclo di vita un Vault deve essere inizializzato una sola volta(Idealmente in fase di inizializzazione del progetto) e aggiornato ogni qualvolta gli envfiles subiscano dei cambiamenti.
+Vediamo di seguito come è possibile effettuare il primo setup ed utilizzarlo.
+
+### Installazione
+
+Se non è ancora stato installato, il primo comando di dotenv-vault su npx lo installerà per noi.
+
+### Inizializzazione di un Vault
+
+Per inizializzare un vault è necessario aprire la command line e digitare:
+`npx dotenv-vault new`
```

Comment:
Questo dovrebbe essere fatto dallo starterkit... Mettere quei comandi

Reply:
Cioè? Sposto all'interno della sezione starter-kit la parte di inizializzazione del vault?

Reply:
la cosa figa sarebbe gestirlo nella creazione dello starterkit ... però non c'è ancora in effetti, quindi direi ok cosi, poi lo cambieremo